Scripts/script-UnEscapeURLs.yml

```diff
@@ -44,11 +44,28 @@ script: |-
         }
         // Not Proofpoint so just un-escape
         if (v === 0) {
-            //Normalize: 1) [.] --> . 2) hxxp --> http 3) &amp --> & 4) http:\\ --> http://
-            var url = input.replace(/\[\.\]/g, '.').replace(/hxxp/i, 'http').replace(/&amp;/g, '&').replace(/http:\\\\/g, 'http://')
-            // Normalize the URL with http prefix
-            if (url.search(/http:/i) === 0 && url.search(/http:\/\//i) !== 0) {
-                url = url.replace(/http:/i, 'http://')
+            var url = input
+            if (input.startsWith('https') || input.startsWith('hxxps')) {
```

Comment:
You sub-scoped the normalization under if blocks - need to normalize in all cases so this should be outside any if block. Otherwise cases like `www[.]example[.]com` wouldn't be normalized as necessary. The other option is to move that bit of the normalization outside of the if blocks.